tests/112-test-use-devs-to-create-container-root-volume.sh

```diff
@@ -27,12 +27,12 @@ CONTAINER_ROOT_LV_MOUNT_PATH=$root_lv_mount_path
 CONTAINER_THINPOOL=container-thinpool
 EOF
 
- # Run docker-storage-setup
- $DSSBIN $infile $outfile >> $LOGS 2>&1
+ # Run container-storage-setup
+ $CSSBIN $infile $outfile >> $LOGS 2>&1
 
  # Test failed.
  if [ $? -ne 0 ]; then
-    echo "ERROR: $testname: $DSSBIN failed." >> $LOGS
```

Comment:
All this naming of DSSBIN to CSSBIN can really be part of a separate PR in preparation to renaming to container-storage-setup. This will reduce size of PR.